HIP-xxx - Pause SP Hex Boosting Rewards For All Radios.md

```diff
@@ -0,0 +1,81 @@
+# HIP-xxx - Pause HIP-86 SP Hex Boosting Rewards For All Radios
+
+- Author(s): [@mrfizzy99](https://github.com/mrfizzy99) (Fizzy99 - Fiz-Tech.net) , [@0lav](https://github.com/0lav) (SaintOlav), [@capjbadger007](https://github.com/capjbadger007) (ElonTusk) , [@bigbuffer](https://github.com/bigbuffer) (JD - Moken) , [Max Gold](https://github.com/GoldHawksAssociates) (Max - GoldHawksAssociates) , [@Gateholder](https://github.com/Gateholder) (Gateholder Worthington Networks)
+- Start Date: 2024/02/13
+- Category: Economic
+- Original HIP PR: <!-- leave this empty; maintainer will fill in ID of this pull request -->
+- Tracking Issue: <!-- leave this empty; maintainer will create a discussion issue -->
+- Vote Requirements: veMOBILE Holders
+
+## Summary
+
+This HIP aims to pause and remove Service Provider Hex Boosting that was passed in HIP-84, to be completely reworked with current available equipment that was not available previously in mind and new economics to better scale nationwide and beyond.
+
+## Prior Related HIPs
+
+HIP-84 created Service Provider Hex Boosting.
+
+
+## Motivation
+
+- As of 02/14/2024, when the Nova Dev team announced that HIP-84 implementation is now complete, it was not correctly done so and omitted the specified requirements of "at least three unique phones with discovery mapping enabled have successfully connected and passed at least 1MB of data at the location of coverage (as evidenced by the Mobile Oracle)."
+
+- Current implementation of HIP-84 also calls for the network to reward boosted hexes out of the networks PoC bucket. However, multiple issues and concerns about this model have arisen that make it detrimental to the core of the network.
+In other words, the economics of HIP-84 is disproportionately skewed, as the it only takes a small amount of MOBILE to be burned by the Service Provider to boost a hex, and it emits many more multiples of the MOBILE for that hex. While already being inherently unfair to the network as a whole, this also leaves the mechanism open for a dishonest Service Provider to boost hexes for their own gain. 
```

Comment:
So afaict the primary concern here is that the multiplier is too large? Why not adjust that and roll forward?

Could you add some details on how you'd see SPs boosting for their own gain and why they'd be incentivized to do so?

Reply:
The ideas for 1:1 Burn to Emitted is the way the I foresee the best way forward, but is for discussion on the revamped 84.  

I am not sure how simple I can explain this.  The mechanism now is BURN MOBILE for a hex, but that BURN is so small to the when compared to the rewards that is rewarded to that hex over that time.   How is burning around 1100 MOBILE for a 100x hex for 6 months, equal and fair to that easily being emitted and rewarded within a day or week?